MATLAB.gitlab-ci.yml

```diff
@@ -3,10 +3,11 @@
 # This specific template is located at:
 # https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/MATLAB.gitlab-ci.yml
 
-# Use this template to run MATLAB and Simulink as part of your CI/CD pipeline. The template includes three jobs:
+# Use this template to run MATLAB and Simulink as part of your CI/CD pipeline. The template includes four jobs:
```

Comment:
Let's replace the first sentence with something we use on other pages, such as here: https://github.com/mathworks/matlab-bamboo-plugin/blob/main/CONFIGDOC.md#:~:text=This%20plugin%20enables%20you%20to%20build%20and%20test%20your%20MATLAB%C2%AE%20project%20as%20part%20of%20your%20Bamboo%C2%AE%20build.

Proposed text:

Use this template to build and test your MATLAB project as part of your CI/CD pipeline. ...